dashboard/app/controllers/reviewable_projects_controller.rb

```diff
@@ -0,0 +1,39 @@
+class ReviewableProjectsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :decrypt_channel_id, only: [:create]
+
+  check_authorization
+  load_and_authorize_resource :reviewable_project, only: [:destroy]
+
+  # POST /reviewable_projects
+  def create
+    authorize! :create, ReviewableProject
+
+    @reviewable_project = ReviewableProject.where(
+      user_id: current_user.id,
```

Comment:
I think this should be derived from the channel ID, not the current user. Current user can be used to decide whether they have access to this action or not.

Reply:
should we verify that the current user and the derived user id from the channel id match?

Reply:
Ah right, well I guess since only the project owner should be able to mark their own project reviewable, yep we'd need to verify against the channel ID derived user; however, if that is the case then project owner should be the same as current user right? Regardless though I'll update this to be project owner for consistency's sake.

Reply:
Yeah I think to Molly's point we should check that current user and channel ID derived user match (in the `authorize!` step) from a security perspective